### PR TITLE
gtk-mac-integration 2.0.5 (new formula)

### PR DESCRIPTION
**Pull request by yesimon**
Formula for gtk-mac-integration.

### CODE REVIEW

File: Library/Formula/gtk-mac-integration.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  sha1 "3b48e85e484b595718c869ac5af6e315f64449d6"
+
+  depends_on "gtk+"
+  depends_on "glib"
+  depends_on "gtk-doc" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on :x11
```
Comment (DomT4) [inline Library/Formula/gtk-mac-integration.rb]: Hard X11 dep may require this to go in ` homebrew-x11 `.

**Comment by yesimon** [comment]
I actually made the formula a while ago so I'm not sure how X11 interacts with superenv and co. How should I check that there isn't a hard dependency on X11? I was just thinking that you probably wouldn't install this formula without X11 because that doesn't make any sense.

**Comment by DomT4** [comment]
> I actually made the formula a while ago so I'm not sure how X11 interacts with superenv and co.

` superenv ` inserts the X11 paths into the various build variables when an X11 dep is declared.

> How should I check that there isn't a hard dependency on X11?

Make the dep ` => :optional ` and see if compile/usage breaks :smile_cat:.

>  I was just thinking that you probably wouldn't install this formula without X11 because that doesn't make any sense.

The GTK stuff is complicated in this regards. @mikemcquaid has long-term thoughts on the GTK/X11 relationship and how to handle it in Homebrew; He'll be the best source for ideas on whether this belongs in Homebrew/homebrew or Homebrew/X11.

File: Library/Formula/gtk-mac-integration.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/gtk-mac-integration.rb]: Yep, sorry.

**Comment by mikemcquaid** [comment]
I think it needs to be in `homebrew-x11`. Longer-term we may have a dedicated GTK tap.

**Pull request closed without merging**